### ISSUE TITLE
Is it alive?

### ISSUE THREAD

**Issue by onfoot** [zulip/zulip-ios#12]
Hi,
if I wanted to make some updates to the iOS app to modernize the code and app's support for newer devices, provided that those changes will be accepted, will they be incorporated into the official App Store release?

**Comment by timabbott** [comment]
Yep!  There's no regular release cadence, so App Store releases happen when there's significant improvements that have been made in git but haven't made it to the App store yet.
